translations/ui-inventory/en.json

```diff
@@ -119,6 +119,9 @@
   "identifier": "Identifier",
   "identifiers": "Identifiers",
   "addIdentifier": "Add identifier",
+  "search.isbn": "- ISBN",
+  "search.issn": "- ISSN",
```

Comment:
Please remove the "-" symbol from the strings.  This type of indentation should be done in code...the only thing that should be in the `en.json` string is the label itself.

Reply:
@dltj thanks for your comments and description in the ticket. The changes were merged to the master branch by @nielserik.
So I will create a new pull request and add you to reviewers.